LEAF_Request_Portal/form.php

```diff
@@ -1371,7 +1371,7 @@ public function getProgress($recordID)
         														AND indicators.disabled = 0
                                                                 AND data != ""', $vars);
 
-        $resCount = $this->db->prepared_query("SELECT categoryID, COUNT(*) FROM indicators WHERE required=1 AND disabled = 0 AND (conditions IS NULL OR conditions = '') GROUP BY categoryID", array());
```

Comment:
https://www.percona.com/blog/2007/04/10/count-vs-countcol/ I had it in my head that count(column) was faster, guess not.